src/components/PhraseCard.js

```diff
@@ -1,27 +1,57 @@
 import styled from 'styled-components';
-import { useState } from 'react';
-import DeleteButton from './DeleteButton.js';
-import HeartButton from './HeartButton.js';
+import React, { useState } from 'react';
```

Comment:
I don't really see, why you need to import `React`? Maybe i just missed it :)

Reply:
thx...no it´s not necessary :D vscode does some strange things